service/schemas/users.js

```diff
@@ -1,8 +1,38 @@
 import mongoose from 'mongoose';
+import bcrypt from 'bcryptjs';
 
 const { Schema } = mongoose;
 
-const user = new Schema({});
+const user = new Schema({
+  name: {
+    type: String,
+    required: true,
+  },
+  email: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  password: {
+    type: String,
+    required: true,
+  },
+});
+
+user.pre('save', async function (next) {
```

Comment:
Takiego zapisu w Schema jeszcze nie widziałem.
Z reguły hashowałem hasło w trakcie rejestracji użytkownika
Czy testowałeś to już w swoim wcześniejszym Repo?
Jeśli śmiga to przechodzi.

Reply:
Jak to mawiał klasyk: "u mnie działa" :D. 
user.pre() to jest taki middleware w mongoose, ale można też w sumie zahashować hasło też w controllerze podczas tworzenia usera. Zobaczymy czy ten sposób nie będzie przysparzał kłopotów.